### PR TITLE
Migrate issue templates to .yml

### PR DESCRIPTION
**Pull request by AlenaSviridenko**
# Description
Improvement:
.yml issue templates provide better control over submitted issues, can set required fields. Also, it is more convenient to submit such issues where each input is a different field

Rendered form:
- 🐓 [Announcement](https://github.com/AlenaSviridenko/virtual-environments/blob/templates-as-yml/.github/ISSUE_TEMPLATE/announcement.yml)
- 🐝 [Bug report](https://github.com/AlenaSviridenko/virtual-environments/blob/templates-as-yml/.github/ISSUE_TEMPLATE/bug-report.yml)
- 🦆 [Tool request](https://github.com/AlenaSviridenko/virtual-environments/blob/templates-as-yml/.github/ISSUE_TEMPLATE/tool-request.yml)

## Check list
- [x] Related issue / work item is attached
- [n\a] Tests are written (if applicable)
- [n\a] Documentation is updated (if applicable)
- [n\a] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/announcement.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    attributes:
+      label: Virtual environments affected
+      multiple: true
+      options:
+        - Ubuntu 16.04
+        - Ubuntu 18.04
+        - Ubuntu 20.04
+        - macOS 10.15
+        - macOS 11
+        - Windows Server 2016 R2
```
Comment (al-cheb) [inline .github/ISSUE_TEMPLATE/announcement.yml:31]: nitpick: Windows Server 2016 R2 -> Windows Server 2016

**Review by al-cheb — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    attributes:
+      label: Virtual environments affected
+      multiple: true
+      options:
+        - Ubuntu 16.04
+        - Ubuntu 18.04
+        - Ubuntu 20.04
+        - macOS 10.15
+        - macOS 11
+        - Windows Server 2016 R2
```
Comment (maxim-lobanov) [inline .github/ISSUE_TEMPLATE/bug-report.yml:28]: nitpick: `Windows Server 2016 R2` -> `Windows Server 2016`

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      description: A clear and concise description of what the bug is, and why you consider it to be a bug.
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Issue type
+      options:
+        - Bug
+        - Question
+        - Feature request
```
Comment (maxim-lobanov) [inline .github/ISSUE_TEMPLATE/bug-report.yml:17]: Do we really need this dropdown? I guess there is a separate template for feature requests

Comment (AlenaSviridenko) [reply .github/ISSUE_TEMPLATE/bug-report.yml:17]: hm, yeah, maybe it is redundant, will remove it then

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+      label: Image version and build link
+      description: |
+        Image version where you are experiencing the issue. Image version can be found in build logs under "Set up job" -> "Virtual Environment" -> "Version".
+        If you have a public example, please, provide a link to the failed build.
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Is it regression?
+      description: If yes, please, provide latest image version where the issue didn't persist, and a link to the latest successful build.
```
Comment (miketimofeev) [inline .github/ISSUE_TEMPLATE/bug-report.yml:36]:
```suggestion
      description: If yes, please, provide the latest image version where the issue didn't persist, and a link to the latest successful build.
```

**Review by maxim-lobanov — DISMISSED** [review]

**Review by miketimofeev — APPROVED** [review]

**Review by maxim-lobanov — APPROVED** [review]

**Comment by lisa1976** [comment]
BUNBURY ANGELS SAY THANK YOU!

File: .github/ISSUE_TEMPLATE/tool-request.md [hunk 5]
```diff
@@ -1,43 +0,0 @@
----
-name: Tool request
```
Comment (MelnikovAG) [inline .github/ISSUE_TEMPLATE/tool-request.md:2]: JOB, CLA

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 6]
```diff
@@ -1,43 +0,0 @@
----
-name: Bug report
```
Comment (MelnikovAG) [inline .github/ISSUE_TEMPLATE/bug-report.md:2]: bug

File: .github/ISSUE_TEMPLATE/tool-request.md (hunk 5 above)
Comment (MelnikovAG) [reply .github/ISSUE_TEMPLATE/tool-request.md:2]: bug

**Pull request merged**